### PR TITLE
Sergio/katas

### PR DESCRIPTION
**Pull request by sergioale9**
I added the katas that i was missing (multiples and likes)

### CODE REVIEW

**Comment by J0c4** [comment]
Hi @sergioale9 , changes look good, please solve conflicts reported and we should be able to merge.

Since we have met the deadline, please do not add any other changes than conflicts resolution, otherwise, PR will be rejected

**Comment by sergioale9** [comment]
@J0c4  I already solved the problem with gradle file, please approve the review.

**Review by J0c4 — APPROVED** [review]

**Comment by sergioale9** [comment]
@J0c4  Jose please approve again, because one of my classmates merge his branch and i had to fix the gradle file again

**Review by J0c4 — DISMISSED** [review]

**Comment by sergioale9** [comment]
i modified the build gradle .....again

File: build.gradle [hunk 1]
```diff
@@ -208,23 +214,19 @@ task runPhonenumberHugo (dependsOn: 'classes', type: JavaExec) {
 
 task runferHighlow (dependsOn: 'classes', type: JavaExec) {
     main = 'jalau.at18.katas.highestlowest.fernanda.Main'
-    standardInput=System.in
```
Comment (J0c4) [inline build.gradle:211]: Hi @sergioale9 , thanks for solving the conflicts, please restore the lines like this so tasks do not get broken

**Review by J0c4 — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (sergioale9) [reply build.gradle:211]: Sorry Jose, I already repair the file with the lines deleted

**Review by J0c4 — APPROVED** [review]

**Pull request merged**